<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>815</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="FuncDimLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>431</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Choose dimension of the vector function&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="NumInputLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Choose number of input variables&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="FuncDimBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>180</y>
      <width>87</width>
      <height>22</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="currentText">
     <string>1</string>
    </property>
    <property name="maxVisibleItems">
     <number>100</number>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="InputDimBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>240</y>
      <width>87</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxVisibleItems">
     <number>100</number>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>601</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;SmartDiff&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Step 1: Dimension specification&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="OKButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="Instructions">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>460</y>
      <width>601</width>
      <height>271</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline; color:#ff0000;&quot;&gt;Instructions to input the function:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;1. Input the variables as x0, x1, x2. In addition to 0-9 and the decimal point, do &lt;span style=&quot; font-weight:600;&quot;&gt;not&lt;/span&gt; put in numerical values pi and e. &lt;/p&gt;&lt;p&gt;2. In addition to +,-,*,/, here are the functions to choose from: &lt;/p&gt;&lt;p&gt;- Power function: f(x)**g(x) should be written as f(x)**g(x)&lt;/p&gt;&lt;p&gt;- Logarithmic function log_n (x) should be written as log(x, n)&lt;/p&gt;&lt;p&gt;- Exponential function e**f(x) can be written as exp(f(x)) or e**f(x)&lt;/p&gt;&lt;p&gt;- Other functions: sqrt, sin, cos, tan, arcsin, arccos, arctan, sinh, cosh, &lt;/p&gt;&lt;p&gt;tanh, logistic are written as such&lt;/p&gt;&lt;p&gt;- You may put in numerical values pi and e&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="ErrMsg">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>340</y>
      <width>591</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>697</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
